corehq/apps/cloudcare/static/cloudcare/js/form_entry/utils.js

```diff
@@ -89,7 +89,13 @@ hqDefine("cloudcare/js/form_entry/utils", function () {
         // Must add the form-control class to the input created by mapbox in order to edit.
         var inputEl = $('input.mapboxgl-ctrl-geocoder--input');
         inputEl.addClass('form-control');
-        inputEl.on('keydown', _.debounce(self._inputOnKeyDown, 200));
+        inputEl.on('keydown', _.debounce((e) => {
+            inputOnKeyDown(e);
+
+            if (e.key === "ArrowUp" || e.key === "ArrowDown") {
+                $("#geocoder-option-sr").html("<p>" + $("ul.suggestions li.active").text() + "</p>");
```

Comment:
Is there a need to remove it once it has been read?
This tag won't come in the path of the screen reader if not through geocoder dropdown?

Reply:
The screen reader reads this tag only once, we don't need to remove it. Yeah it wont be in the path of screen-reader, only geocoder changes trigger this.

Reply:
Thanks